pkg/installmanager/installmanager.go

```diff
@@ -950,6 +1003,70 @@ func (m *InstallManager) generateAssets(cd *hivev1.ClusterDeployment, workerMach
 	return nil
 }
 
+func patchLabelMachineSetManifest(manifestBytes []byte, poolsByType map[string]string, logger log.FieldLogger) (*[]byte, error) {
```

Comment:
I want to see this DRYed with the other patch func eventually. But we can't do that until we've cut both over to using gjson/sjson. So for now please just add a TODO comment and reference the card you created for that cutover.